bazel/repositories.bzl

```diff
@@ -795,6 +795,10 @@ def _proxy_wasm_cpp_host():
     external_http_archive(
         name = "proxy_wasm_cpp_host",
         build_file = "@envoy//bazel/external:proxy_wasm_cpp_host.BUILD",
+        # The patch fixes WASM on s390x https://issues.redhat.com/browse/MAISTRA-2648
+        # The permanent fix is https://github.com/proxy-wasm/proxy-wasm-cpp-host/pull/198
+        patches = ["@envoy//bazel/external:proxy_wasm_cpp_host.patch"],
```

Comment:
nit: Generally speaking it's a good idea to name the file after its meaning or purpose, or even with the title of the original patch, instead of naming it with the name of the package. What if, in the future there is another patch for this same package?